Main.java

```diff
@@ -1,76 +1,64 @@
 package com.company;
 
 import java.io.*;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.util.StringTokenizer;
+
+import com.company.dbconnection.DatabaseLink;
 
 public class Main {
 
-    final static String FILE_TYPE_TXT = ".txt";
-    final static String FILE_TYPE_JSON = ".json";
-    final static String REMOVE_SPECIAL_CHAR = "\\p{Punct}";
-    final static String SINGLE_SPACE = "";
+    private static DatabaseLink databaseLink = new DatabaseLink();

```

Comment:
Why is this static? Any reason?

Reply:
For the first three, since they are static, we don't need to create an instance of the class to access it. We can just access them using the classname itself.  databaseLink is used in multiple methods within the class, hence it is declared as a class variable.